components/NewRelicAgent/NRAgent.brs

```diff
@@ -1427,7 +1438,7 @@ function nrStateTransitionPlaying() as Void
         shouldSendStart = m.nrIsInitialBuffering
         nrSendBufferEnd()
         
-        if m.nrVideoObject.position = 0
+        if m.nrVideoObject.position = 0 or m.nrShouldTrackResumeAsContentStart
```

Comment:
Suggestion for New Relic folks:
It might be cleaner to remove the `position = 0` check. I don't see this check being performed in `newrelic-video-core`.

Reply:
@asllop

Reply:
@sathya-sundar have you tried @timalacey's solution? That should fix the issue without the need of introducing the `nrShouldTrackResumeAsContentStart` flag.

Reply:
@sathya-sundar this [PR](https://github.com/newrelic/video-agent-roku/pull/73) should fix your issue. Could you test it and confirm?